the-content-guide/readme.md

```diff
@@ -7,14 +7,100 @@ The target audience are communication professionals. See it as a comprehensive g
 
 <img src="/images/contentcat.jpg" width="150">
 
-## File storage
+## Intro
 
-Each chapter should be it's own file. When you add a chapter, please provide the name of the file with a number. This number represents the order of the document chapters. After the number comes the chapters' name.
+Why do you need content marketing or branded content? Content marketing is a way to create a bond between a reader and a company. By creating content the reader will thoroughly enjoy, the reader will associate the brand with an awesome feeling and automatically think about that company first whenever searching for a product/service in that domain.
 
-## Pull requests
+Example: A solar cell company creates blogposts about:
+* how a solar cell work  
+* a comparison between green energy products
+* where is the future of solar cell power heading?
+* a foto collage of employees risking their lives on roofs
 
-Make your own branch. When working on the document, create a pull request for each document.
+So how do you attract readers and keep them engaged? There is an increasing demand for *'high quality content'*. But what is high quality content? What ingrediënts make a nice meal that people will remember? Better yet, how do you get people to ask for and share the recipe?
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, how to create content you can be proud of!
```

Comment:
The last sentence is a fragment. You can either combine it with the previous sentence or add something like: "Most of all, this guide will tell you how to..."